### PR TITLE
Use local FDB config rather than remote config

### PR DESCRIPTION
**Pull request by victoria-cherkas**
Relates to https://meteoswiss.atlassian.net/browse/APNRZ-430

### CODE REVIEW

File: notebooks/2_Precompute_and_Store_Echotop_to_FDB.ipynb [hunk 1]
```diff
@@ -51,13 +51,14 @@
...
-    "store: remote\n",
-    "\"\"\"\n",
-    "os.environ[\"FDB_REMOTE_RETRIEVE_QUEUE_LENGTH\"] = \"100\"\n"
+    "type: local\n",
+    "engine: toc\n",
+    "schema: /scratch/mch/vcherkas/fdb-realtime-lcm/schema\n",
+    "spaces:\n",
+    "- handler: Default\n",
+    "  roots:\n",
+    "  - path: /scratch/mch/vcherkas/fdb-root-realtime\n",
```
Comment (cfkanesan) [inline notebooks/2_Precompute_and_Store_Echotop_to_FDB.ipynb:60]: Is it expected for users to be able to write to this location?

Comment (victoria-cherkas) [reply notebooks/2_Precompute_and_Store_Echotop_to_FDB.ipynb:60]: Should be the case that anyone in group s83 can write to it. Will add a comment

Comment (victoria-cherkas) [reply notebooks/2_Precompute_and_Store_Echotop_to_FDB.ipynb:60]: Can you test whether the write back to FDB works for you?

Comment (cfkanesan) [reply notebooks/2_Precompute_and_Store_Echotop_to_FDB.ipynb:60]: It doesn't work for me. I got a permission denied error.
```
FDBException: Error in function fdb_archive_multiple: Failed system call: fd_ = ::open( tocPath_.localPath(), iomode, mode_t(0777) ) in  (/users/ckanesan/spack-c2sm/spack-build_stage/spack-stage-fdb-5.11.116-25exmiavnxzmi2kyswfatqyx6srxiige/spack-src/src/fdb5/toc/TocHandler.cc +729 writeInitRecord)  (Permission denied) [/scratch/mch/vcherkas/fdb-root-realtime/20240626:1200:enfo:od:0001:icon-ch1-eps:ememb:1/toc]
```

**Review by cfkanesan — APPROVED** [review]

**Pull request merged**